pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [tool.black]
 line-length = 79
 skip-string-normalization = true
-target-version = ['py27', 'py35', 'py36', 'py37', 'py38']
+target-version = ['py35', 'py36', 'py37', 'py38']
```

Comment:
Pyramid 1.10 still supports Python 2.7. I don't think we can drop it. Let's see what maintainers say.

Reply:
It's ok to drop it.